cmd/broscoli/main_test.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/hypebeast/go-osc/osc"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+)
+
+const (
+	SubCmdFlags = "SUB_CMD_FLAGS"
+)
+
+func TestAPI(t *testing.T) {
+	_ = os.Remove("/tmp/foo")
+	file, _ := os.Create("/tmp/bar")
+	defer file.Close()
+	tests := []struct {
+		name            string
+		actions         map[string]*Action
+		wantFileToExist bool
+		wantErr         bool
+	}{
+		{
+			name: "creating a file",
+			actions: map[string]*Action{"/create": {
+				Command:    "/bin/touch /tmp/foo",
```

Comment:
These should the use the temp file names. I added file names with spaces to proof that the shell argument line logic works. If would be cool to have this here as well.